README.md

```diff
@@ -1,12 +1,11 @@
-open-orchestra-media-bundle
+open-orchestra-workflow-bundle
```

Comment:
In my mind, workflow features are very interrelated and not easily separable. It will not possible to get fonction feature without getting role. When the profile feature will be implemented, it will not possible to get it without getting fonction feature.